packages/nitro-protocol/contracts/AssetHolder.sol

```diff
@@ -235,6 +235,8 @@ contract AssetHolder is IAssetHolder {
         );
         uint256 initialHoldings = holdings[fromChannelId];
 
+        require(initialHoldings > 0, 'no holdings for this channel');
```

Comment:
What's the gas cost of the additional statement?

We don't expect to hit this revert. So, i could argue that you're requiring every user to pay a small amount of gas for a line of code that in theory never triggers a revert, and also doesn't provide any additional safety.

Reply:
(meta comment: this highlights the benefit of the gas accounting scripts)

Reply:
The additional statement costs 17 gas, according to my investigations (plus a bigger increase to deployment costs). 

I agree that it doesn't provide any additional safety, and that we don't expect to hit this code path. I hit the code path while writing a gas benchmarking script, which this change helped me debug.

I think on balance maybe we just close this, or replace the change with a comment.